### PR TITLE
open-cluster-management/backlog#17882 Add AIS

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_infra_env.adoc [hunk 1]
```diff
@@ -14,10 +14,88 @@ You need the following prerequisites before creating an infrastructure environme
...
+* A configured instance of the Central Infrastructure Management (CIM) feature on your hub cluster. See xref:../clusters/create_infra_env.adoc#enable-assisted-installer[Enable assisted installer] for the procedure.
 * Your {ocp-short} https://console.redhat.com/openshift/install/pull-secret[pull secret]. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/images/managing-images#using-image-pull-secrets[Using image pull secrets] for more information. 
 * Your SSH key that is in your `~/.ssh/id_rsa.pub` file, by default.
 
+[#enable-assisted-installer]
+== Enabling the Assisted Installer Service
+
+The Assisted Installer Service (AIS) is provided with {product-title-short} and deploys {ocp-short} clusters. AIS is deployed when you enable the MultiClusterHub Operator on the {product-title-short} hub cluster, but must be enabled.
+
+For distributed units (DUs), {product-title-short} supports {ocp-short} deployments that run on a single bare-metal host. The single node cluster acts as both a control plane and a worker node.
```
Comment (jelkosz) [inline clusters/create_infra_env.adoc]: I would skip the DU part - its a very telco specific term. Actually I would skip this whole section - it is not related to the enabling of the AIS

File: clusters/create_infra_env.adoc [hunk 2]
```diff
@@ -14,10 +14,88 @@ You need the following prerequisites before creating an infrastructure environme
...
+
+To enable the Assisted Installer Service, complete the following steps: 
+
+. Modify the `HiveConfig` resource to enable the feature gate for Assisted Installer by running the following command:
++
+----
+oc patch hiveconfig hive --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"}}}'
```
Comment (jelkosz) [inline clusters/create_infra_env.adoc:32]: Please note this will not be needed for 2.4.1, it is still needed for 2.4.0 though.

**Comment by jelkosz** [comment]
Just two small comments, other than that looks pretty good!
@filanov can you plz check too?

File: clusters/create_infra_env.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/create_infra_env.adoc]: Removed the paragraph about DUs.

File: clusters/create_infra_env.adoc (hunk 2 above)
Comment (cadawson) [reply clusters/create_infra_env.adoc:32]: Got it. We can remove when 2.4.1 releases.

**Review by cadawson — COMMENTED** [review]
Thanks @jelkosz . Comments addressed.

**Comment by filanov** [comment]
> Just two small comments, other than that looks pretty good! @filanov can you plz check too?

Is there a troubleshooting section?
For example after the deployment user can look at the status or at the pods that are created etc.

**Comment by cadawson** [comment]
Thanks @filanov. We don't have specific troubleshooting information for that section, but we have an overall product troubleshooting section. I am not sure that what you are suggesting requires a troubleshooting topic. Would it work to add content at the end of the section like:

You can view the status of your cluster by viewing it on the Clusters page in the console or by checking the status of the pods by running the following command: 
oc get pods

Would something like that work?

**Comment by filanov** [comment]
It could help, @djzager @celebdor maybe you have more points to add?

**Comment by djzager** [comment]
I suppose verifying that the assisted-service and assisted-image-service deployments are ready could be helpful but I can't think of any meaningful additional points to mention.

File: clusters/create_infra_env.adoc [hunk 3]
```diff
@@ -14,10 +15,86 @@ You need the following prerequisites before creating an infrastructure environme
...
+oc create -f agent_service_config.yaml
+----
++
+The output might resemble the following example:
++
+----
+agentserviceconfig.agent-install.openshift.io/agent created
+----
+
+Your CIM service is configured. Continue with xref:../clusters/create_infra_env.adoc#creating-your-infra-env-with-the-console[Creating your infrastructure environment with the console].
```
Comment (jelkosz) [inline clusters/create_infra_env.adoc]: Just following up on @djzager 's comment, I'd added here something like:
Your CIM service is configured. You can verify it is healthy by checking the assisted-service and assisted-image-service deployments and their pods are ready and running. Continue with xref:../clusters/create_infra_env.adoc#creating-your-infra-env-with-the-console[Creating your infrastructure environment with the console].

Comment (cadawson) [reply clusters/create_infra_env.adoc]: Updated the content. Look ok @jelkosz ?

**Comment by jelkosz** [comment]
looks good to me!

**Comment by cadawson** [comment]
Thanks @jelkosz . I will work on getting this in the refresh today. If you see anything that needs to change, feel free to open an issue. We can change it.

**Pull request merged**